### ISSUE TITLE
IdeaOutline - Dialogue Mapping for Collaboration, Decisions, Tasks, Projects, Organizations & Governance

### ISSUE THREAD

**Issue by IdeaOutline** [JoinColony/colonyHackathon#87]
### Colony Hackathon Submission

**Project Title**

# **IdeaOutline - Dialogue Mapping for Collaboration, Decisions, Tasks, Projects, Organizations & Governance**
---

**Project Description**

An intuitive, user friendly platform enables **dialogue mapping** in the form of a hierarchy tree outline with infinite threading and collapsible branches

Participants are guided by a built in collaboration process and make relevant, rational comments, while spam, uncivil and irrational comments get filtered out.

Consequently, a group can communicate efficiently, find the best answers, come to unanimous consensus, self-organize, lead themselves, create actionable events and achieve the best of all possible results.

Specific processes can be uploaded into IdeaOutline so groups can follow along and achieve specific results.

A few practical use examples:

Education
Students can collaborate together online and do their homework, or something more involved like a class project. 

Science
A group of scientists working remotely can create a hypothesis and research process, conduct a study, and with the results, write a paper that conforms to the rigors of the scientific method and peer review.

Startup Organizations
An individual can startup a new organization around a particular purpose. The founder follows a proven two step startup process to attract like minded participants, and then follow an agenda that turns a crowd of people into a competent and productive team.

Politics
Concerned citizens in a voting district can create opportunities to improve any issue, come to consensus, and write policies and bills for their representatives to vote into law.  Even though participants may be bitterly polarized politically, they can address the most wicked of issues and find solutions that lead to consensus.

At first glance, this may all seem improbable. However, all of the processes used in IdeaOutline are being used already in real world situations. Every process has been proven to work. What has not been proven is the combination of them together in this fashion for online collaboration, which is what this prototype hopes to achieve.

The actual interface is quite simple to create, since it consists of a block of repeatable functionality. Then it becomes a matter of tying it into Colony.
Depending on how much time we have left will determine how much additional functionality we add beyond the minimum viable product.

I'm looking for participants. 

**Project Repository**

TDB - New Github project coming soon

**Team Members and Contact info**

kevinschmidt@email.com

**Comment by jrmoreau** [comment]
Super cool idea! @IdeaOutline will you be taking on additional team members?

**Comment by ninjasun** [comment]
really curious to see the result. IMO is impossible. :)

**Comment by IdeaOutline** [comment]
Yes, I need additional team members to complete the project. Please add the tag.
Thanks!

**Comment by IdeaOutline** [comment]
ninjasum, 
I can assure you it is possible, because the processes I use are already being used successfully elsewhere, either online or in real life venues. They're just not being used online together.

**Comment by gichiba** [comment]
Hey @IdeaOutline I really like this idea, and I think it's a perfect use case for Colony. The structure of a dialogue map (as far as I understand it) fits directly within the same structure as the domains and skills hierarchy, which means that all the other elements (such as tasks and reputation) fit as well! 

in the current version of the contracts, however, domains are restricted to a single level, which might impede your ability to make a 'functioning' prototype work. Skill tags, however, behave in much the same way as domains. So, if you're just after making a proof of concept, running the 'Meta Colony' on a private chain, you'd be able to have 'nested' parts of the dialogue map do roughly the same job, but using your own special version of the Meta Colony's global skill tree. 

I hope this helps a little!

**Comment by IdeaOutline** [comment]
Thanks gichiba. Your advise will certainly save time knowing that before we get started.
And thanks for adding the new teammates label.

**Comment by IdeaOutline** [comment]
I still need partners.

I have built a prototype for IdeaOutline using:
https://webflow.com/

I want to use the Colony Network to verify users, and to create an Idea Token. The tokens will be distributed, mainly as awards , to participants according to the value they bring to the IdeaOutlines.

**Comment by jrmoreau** [comment]
@IdeaOutline what kind of team members do you need to join up? Front end? Solidity? Designers? Let us know and we'll help amplify.

**Comment by IdeaOutline** [comment]
@msbcoder and any other coders with time available, I am still looking for team members.

**Comment by IdeaOutline** [comment]
I was not able to get a coder to assist me with my project and so it is now closed.
